### DISCUSSION TITLE
ScriptReference/ParticleSystem.Play

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#42] [Notes]
# ScriptReference/ParticleSystem.Play

https://docs.unity3d.com/ScriptReference/ParticleSystem.Play.html

**Comment by vertxxyz** [comment]
Where appropriate use the overload `withChildren` as `false` to avoid Unity trying to get child ParticleSystems. If you are often calling this with children, it's advised to cache the children yourself and set `withChildren` as `false`.

**Comment by docscomments** [comment]
**Comment by `@homemech`:**  

[Here](https://youtu.be/I9x7FYqocW0) is an example of ParticleSystem.Play with visual scripting in context with other effects.
